packages/pigeon/platform_tests/alternate_language_test_plugin/android/build.gradle

```diff
@@ -57,7 +57,3 @@ android {
         testImplementation "org.mockito:mockito-core:5.1.1"
     }
 }
-
-project.getTasks().withType(JavaCompile){
-    options.compilerArgs.addAll(["-Xlint:all", "-Werror"])
-}
```

Comment:
This was here instead of in the example build.gradle because unlike with all of our actual plugins we don't need to worry about breaking clients, and so it was easier to just have it here. But now that we're enforcing uniformity, it's easier to make it match the rest of the plugins instead of carving out an exception.